app.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dogstack example",
+  "description": "the example basic dogstack app",
+  "repository": "https://github.com/root-systems/dogstack-example",
+  "addons": [
+    "heroku-postgresql:hobby-dev",
+    "papertrail"
+  ],
+  "env": {
+    "APP_URL": {
+      "description": "the url that the app is running at (if you are deploying from the README, replace [app name] below with the app name you chose above)",
```

Comment:
@iainkirkpatrick It’s not clear what was called “[app name] above”

I’d also suggest adding some simple instructions to the reader file by the button stepping a new user through getting an app name and editing this app.json file before clicking the button.

Reply:
@agentlewis do you mean “with the app name you chose above” is unclear? That refers to the form you get taken to when you click the button - is it unclear in the context of that form? 

The user shouldn’t have to edit anything before clicking the button - when they click and get to the form, they just make up a name for their app

Or am I misunderstanding you maybe?

Reply:
Ah sorry I didnt understand that the button took you to a form. I get it now.

Maybe just say the heroku app name. “Above” made me feel I missed something.